scripts/generate_refcats_gen3.py

```diff
@@ -34,7 +34,7 @@
 
 import lsst.log
 import lsst.sphgeom
-from lsst.daf.butler import Butler, CollectionType, DatasetRef, DatasetType, FileDataset
+from lsst.daf.butler import Butler
 
 
 logging.basicConfig(level=logging.INFO, stream=sys.stdout)
```

Comment:
Consider turning on VERBOSE since that is what transfer_from uses. (you need to get VERBOSE from `lsst.utils.logging` if you decide to use it)

Also, is the line below needed? Nothing in butler is going to issue an `lsst.log` log message.

Reply:
Is there a particular reason to use VERBOSE? Otherwise, I'm inclined to stick to INFO; I've found that a lot of LSST code over-logs to the point where it's hard to tell what's going on, and status reports should be all that the user needs here.

Happy to remove the MDC line; I think it was cargo-culted in.

Reply:
You can turn on VERBOSE temporarily and see what it does. It's up to you.